include/RadioPropa.h

```diff
@@ -17,6 +17,7 @@
 #include "radiopropa/Referenced.h"
 #include "radiopropa/Source.h"
 #include "radiopropa/ScalarField.h"
+#include "radiopropa/IceModel.h"
```

Comment:
Is there a good reason for separating IceModel and Scalarfield?

Reply:
It did it for transparency